sympy/functions/special/delta_functions.py

```diff
@@ -409,25 +409,23 @@ class Heaviside(Function):
 
     Explanation
     ===========
-
+Heac
     Heaviside function has the following properties:
```

Comment:
There seems to be a spurious change here

Reply:
Thanks, removed.